dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/RelationshipOperationParamsMapper.java

```diff
@@ -54,9 +56,21 @@ class RelationshipOperationParamsMapper {
 
   @Transactional(readOnly = true)
   public RelationshipQueryParams map(@Nonnull RelationshipOperationParams params)
-      throws NotFoundException, ForbiddenException {
+      throws NotFoundException, ForbiddenException, BadRequestException {
+
+    if (params.getType() == null) {
+      if (CollectionUtils.isEmpty(params.getRelationships())) {
+        throw new BadRequestException(
+            "Either one between Relationships or TrackerType must be provided");
+      }
+      return RelationshipQueryParams.builder()
```

Comment:
if `RelationshipQueryParams` is internal and all we do is a call like this we don't need a builder but a plain old constructor